src/Repository/AssignmentRepository.php

```diff
@@ -99,9 +99,9 @@ public function insertMultipleNatively(AssignmentDtoCollection $assignments): vo
         $queryParts = [];
         foreach ($assignments as $assignmentDto) {
             $queryParts[] = sprintf(
-                "('%s', %s, '%s', '%s', %s)",
+                "('%s', '%s', '%s', '%s', %s)",
                 (string)$assignmentDto->getId(),
-                $assignmentDto->getUserId(),
+                (string)$assignmentDto->getUserId(),
                 (string)$assignmentDto->getLineItemId(),
```

Comment:
Same as on assignment PR, you don't need to cast it here

Reply:
Fixed